Config.uk

```diff
@@ -1,13 +1,9 @@
 menuconfig LIBCOMPILER_RT
-    bool "compiler-rt - runtime support"
-    select LIBPOSIX_SYSINFO
-    select LIBUKMMAP
-    select LIBUNWIND
-    default n
+	bool "compiler-rt - Compiler runtime support"
+	select LIBPOSIX_SYSINFO
+	select LIBUNWIND
+	default n
 
 if LIBCOMPILER_RT
```

Comment:
Is there a reason this `if` condition is still present? It doesn't seem to do anything.